content/news/2023_11_09_Student_Grants_announced.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Apply Now for Student Grants"
+date: 2023-11-09T13:12:00+05:00
+layout: "news"
+author: Anna Petrasova, Vaclav Petras
+---
+
+<a href="/images/news/student_grants_2023.jpg">
+  <img src="/images/news/student_grants_2023.jpg"
+   alt="GRASS GIS Student Grants"
+   title="Paid opportunity for students to contribute to GRASS GIS"
+   width="45%" style="float:right;padding-left:25px;padding-top:15px">
+</a>
+
+We would like to announce a unique [**paid opportunity**](https://grasswiki.osgeo.org/wiki/Student_Grants) for students to contribute to GRASS GIS!
+GRASS GIS will offer a number of student grants for projects that include development of GRASS documentation, tests, new features or geospatial tools and bug fixing. Check the suggested topics on the [Student Grant wiki](https://grasswiki.osgeo.org/wiki/Student_Grants#Topics).
+
+
+### Why to apply?
+
+**Experience:** Gain hands-on experience in a thriving open-source community.<br>
+**Mentorship:** Work alongside experienced developers who will guide you throughout your journey.<br>
+**Contribution:** Your work will directly enhance the functionality and accessibility of GRASS GIS for users worldwide.<br>
+**Stipend:** Get up to 1000 USD.
+
+
+### How to apply?
+Express your interest in applying through the current GRASS community channels and identify a topic matching your interest. Look for ways to contribute to GRASS GIS that are related to the selected topic and submit a pull request to demonstrate your skills. Draft a proposal and share it to get feedback. For details and eligibility rules, check the [wiki](https://grasswiki.osgeo.org/wiki/Student_Grants).
```

Comment:
Where should candidates share their proposal?

Reply:
That's on the wiki, so we can clarify it there if needed.